README.md

```diff
@@ -17,9 +17,11 @@ This project comprises primarily four crates:
 - The [`ibc-relayer-cli`][relayer-cli-crate-link] is a CLI (a wrapper
   over the `ibc-relayer` library), comprising the
   [`hermes`](https://hermes.informal.systems) binary.
-- The [`ibc-proto`][ibc-proto-crate-link] is a library with Rust types generated from .proto definitions
+- Crate [`ibc-proto`][ibc-proto-crate-link] is a library with Rust types generated from .proto definitions
```

Comment:
Prefixing the crate name with `Crate` sounds weird to me, I would just leave the name. It's common to talk about crates or libraries using just their names without a prefix.

Reply:
Or, to be consistent with what's above:

```suggestion
- The [`ibc-proto`][ibc-proto-crate-link] crate is a library with Rust types generated from .proto definitions
```